test/test_beam_search.py

```diff
@@ -0,0 +1,156 @@
+import unittest
+
+import _dynet as dy ; dyparams = dy.DynetParams() ; dyparams.set_random_seed(13); dyparams.init()
```

Comment:
This is not the preferred way of doing this anymore. Could you please try the new way?
http://dynet.readthedocs.io/en/latest/python_ref.html#dynet.DynetParams

Reply:
I updated it everywhere except here, will that be alright?
https://github.com/neulab/xnmt/pull/175/files#diff-d11c4a7987fb2f1114fff16beb26c6ffR163

Reply:
Hm, I got a duplicate initialization warning so I removed dyparams.init() in xnmt_run_experimnts.py.
Hope this is the correct way to do it:
```
import _dynet
dyparams = _dynet.DynetParams()
dyparams.from_args()
```